project/Dependencies.scala

```diff
@@ -293,14 +293,15 @@ object Dependencies {
         jacksonCore,
         jacksonAnnotations,
         jacksonDatabind,
-        jacksonScala,
         jacksonJdk8,
         jacksonJsr310,
         jacksonParameterNames,
         jacksonCbor,
         lz4Java,
         Test.junit,
-        Test.scalatest)
+        Test.scalatest) ++
+    (if (getScalaVersion() == scala3Version) Seq.empty
+    else Seq(jacksonScala))
```

Comment:
Sneaky :) - how optional is this dependency in practice?

Reply:
It's specific jackson stuff for Scala SDK types, so I guess a user app could still use Jackson with Scala as long as you don't have options, collections etc in the types you want to serialize. (you likely do). We don't require it for any internals though.

I don't see this as the end state, just a way to get it compiling until there is a jackson scala module for Scala 3